src/v1/driver.js

```diff
@@ -59,7 +59,11 @@ class Driver {
       this._createConnection.bind(this),
       this._destroyConnection.bind(this),
       this._validateConnection.bind(this),
-      config.connectionPoolSize
+      {
```

Comment:
What do you think about adding a dedicated `PoolConfig` class for this object?
Then it will be possible to specify type of `config` constructor parameter in `Pool` in JSDoc and get autocompletion from Intellij.